atk/lib/atk.rb

```diff
@@ -11,11 +27,24 @@
   require 'atk.so'
 end
 
-if vendor_dir.exist?
```

Comment:
We shouldn't remove the check.
Because atk gem doesn't depend on gobject-introspection gem on non Windows environment. If it's removed, `require "atk"` raises `LoadError on `gem install atk` only environment (no `gem install gobject-introspection` environment).